privacy_gdpr/tests/test_privacy_gdpr.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp.tests.common import SingleTransactionCase
+from ..wizards.res_partner_gdpr import FIELDS_GDPR
+
+CLEANUP_STRING = '*** GDPR removed ***'
+
+
+class TestPrivacyGdpr(SingleTransactionCase):
+
+    post_install = True
```

Comment:
@hbrunn I think I might disagree with you here. I think it better to run test after all modules have been installed, because it will be more like testing a real live system. If a test fails because of another module installed later, it shows up an incompatibility that would otherwise be missed.

Reply:
Current real use case in Travis CI (where first all modules are installed and then all are upgraded with test-enable flag) for using post_install instead of at_install is:

* A module puts a required field in any general model with a default. The SQL constraint is created.
* Your module is loaded before the other module is loaded, and you try to create a record of that model.
* You will get a not null constraint as the default is not applied.

We put in all tests `at_install=False` and `post_install=True` due to this.

Any way, indeed you should deactivate `at_install`, or you will have the above problem, and in the best case, tests will be run twice.

Reply:
Still need the at_install=False as suggested by Pedro.